components/RootHomePage/LiveStream.js

```diff
@@ -1,118 +1,31 @@
 import React from 'react';
 import styled from 'styled-components';
 import loadable from '@loadable/component';
-import ContentSection from '../shared/ContentSection';
-import LinkButton from '../shared/LinkButton';
-import { above, below } from '../../utilities';
+import SessionHighlight from '../shared/SessionHighlight';
 
 const ReactTwitchEmbedVideo = loadable(
   () => import('react-twitch-embed-video'),
   { ssr: false },
 );
 
-const twoColBp = 'larger';
-
-const Main = styled.div`
-  display: flex;
-  flex-direction: row;
-  margin: auto;
-  justify-content: center;
-
-  ${below[twoColBp]`
-    flex-direction: column;
-  `};
-`;
-
-const SideDetail = styled.div`
-  display: flex;
-  flex-direction: column;
-  flex-grow: 2;
-
-  ${above[twoColBp]`
-    margin-right: 4rem;
-  `};
-
-  ${below[twoColBp]`
-    align-items: center;
-  `};
-`;
-
-const VideoBlock = styled.div`
-  min-width: 90rem;
-  max-height: 55rem;
-  padding-top: 4rem;
-  padding-bottom: 1rem;
-
-  ${below[twoColBp]`
-    min-width: 80%;
-    height: 50rem;
-  `};
-`;
-
-const StyledH3 = styled.h3`
-  color: ${({ theme }) => theme.colors.fonts.light};
-  margin: 3rem 0;
-`;
-
-const Links = styled.ul`
-  list-style-type: none;
-  padding: 0;
-  margin: 0;
-
-  li {
-    float: left;
-    color: ${({ theme }) => theme.colors.fonts.light};
-
-    &:first-child {
-      padding-right: 4rem;
-
-      ${below.xsmall`
-        padding-right: 0;
-      `};
-    }
-  }
-`;
-
 const LiveStream = ({ className }) => {
+  const description =
+    'Broadcasting every Monday and Wednesday, THAT.Live is, well, live! Over on our Twitch channel, community members collaborate with host Clark Sell and share their knowledge and best practices for all things technology and not technology. We invite you to watch and listen in. Viewers with a Twitch account are encouraged to become part of the broadcast by commenting and asking questions in real time using the chat feature. Miss a broadcast? Past live stream videos are available to watch too!';
```

Comment:
Consistency on the casing of .Live/.live.  @astralbodies brought this up as a bug I addressed in in the root landing PR which should be merged in to here.

Reply:
Thanks for catching! Hmm... ok, let me fix this here for now cause I did some refactoring and not sure the rebase with catch it. Once we get all of our PRs merged in to the one-day I can rebase it with the root-landing so our next move can be merge those together.